apps/src/music/blockly/MusicBlocklyWorkspace.js

```diff
@@ -78,8 +79,8 @@ export default class MusicBlocklyWorkspace {
       grid: {spacing: 20, length: 0, colour: '#444', snap: true},
       theme: CdoDarkTheme,
       renderer: experiments.isEnabled('zelos')
-        ? 'cdo_renderer_zelos'
-        : 'cdo_renderer_thrasos',
+        ? Renderers.ZELOS
+        : Renderers.DEFAULT,
```

Comment:
Calling out for Music Lab team (@breville @sanchitmalhotra126) that we recently updated our other labs to also use Thrasos as the default renderer. Thrasos is likely going to remain a firm requirement for Sprite Lab due to Alice's awesome work in this PR.